### PR TITLE
Made menu item names and dialog names consistent 

### LINKED ISSUE #6710: Inconsistencies between menu item names and dialog names
Here is a link to the google doc of the list of Inconsistencies I have found between the Menus and Dialog names. (https://docs.google.com/spreadsheets/d/1Mm7q_BvLJ3bENA1iWqzDdNr9KMeZv6a1/edit?usp=sharing&ouid=109004489783210984776&rtpof=true&sd=true)

While I think this is a good start, I am by no means claiming that it is comprehensive. It is just those that were highlighted by inconsistencies in the French.

There are also certainly instances where discrepancies between the Menu items and Dialog names seem logical and I have therefore not mentioned:
When the menu item needs to be shorter
When the menu steps add information (eg Menu: Describe - Two variable - Graph; opens dialog: Two Variable Graph)
Certain times when the menu item opens to a differently named dialog which is opened in a certain setting which enables the user to complete that task (eg menu Climatic - Dates - Generate Dates; opens dialog Regular Sequence on the Dates Tab).

### PR DESCRIPTION
**Pull request by Gioche6**
Fixes #6710

@africanmathsinitiative/developers , This is ready for review
@rachelkg and @Patowhiz , I have made all the changes listed on the  [Inconsistencies between Menu and Dialog names.xlsx](https://github.com/africanmathsinitiative/R-Instat/files/7156493/Inconsistencies.between.Menu.and.Dialog.names.xlsx)

@lloyddewit , I have changed the text descriptions as recommended. However, the text is not changing in run time because of the translation issue. I think this can be merged, but the issue to remain open until the translation issue is fixed, in which,  it will automatically change.

### CODE REVIEW

**Comment by lloyddewit** [comment]
image
 @Gioche6 For consistency, please could you also correct the capitalization here (e.g. make 'Non-Numeric')? Thanks

File: instat/dlgDeleteRowsOrColums.resx [hunk 1]
```diff
@@ -460,7 +460,7 @@
     <value>CenterScreen</value>
   </data>
   <data name="$this.Text" xml:space="preserve">
-    <value>Delete Rows/Columns</value>
+    <value>Delete Rows or Columns</value>
```
Comment (lloyddewit) [inline instat/dlgDeleteRowsOrColums.resx:463]:
```suggestion
    <value>Delete Columns or Rows</value>
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@Gioche6 Thank you for the changes. I just have a couple of small suggestions. 
I also updated the database and pushed the new database to this PR. This should allow you to test your changes.
I noticed that you have merge conflicts on 'frmMain'. You may need to make a new PR for this.

**Comment by lloyddewit** [comment]
> 
> 
> image
> @Gioche6 For consistency, please could you also correct the capitalization here (e.g. make 'Non-Numeric')? Thanks

@Gioche6 Please could you also resolve the above? thanks

**Comment by Gioche6** [comment]
@lloyddewit , I have made the change you requested.

**Comment by Gioche6** [comment]
@lloyddewit , I have resolved to do the changes in a separate PR, since this one has merging issues.

**Comment by lloyddewit** [comment]
replaced by PR #6771

**Pull request closed without merging**